src/commands/config/setapikey.js

```diff
@@ -31,6 +31,19 @@ class SetAPIKey extends Command {
 		)
 		const apikey = args[0]
 
+		try {
+			const community =
+				await this.client.fagc.communities.fetchOwnCommunity(null, {
+					apikey: apikey,
+				})
+			if (community.contact !== message.author.id)
```

Comment:
We discussed adding a notice for when this happens. Are you still planning to do this?

Reply:
Oops, added.